nekoyume/Assets/_Scripts/UI/Combination/SubRecipeView.cs

```diff
@@ -103,7 +103,10 @@ public void SetData(SheetRow<int> recipeRow, List<int> subrecipeIds)
                 title = resultItem.GetLocalizedName(true, false);
 
                 var stat = resultItem.GetUniqueStat();
-                statText.text = string.Format(StatTextFormat, stat.Type, stat.ValueAsInt);
+                var statValueText = stat.Type == StatType.SPD
+                    ? (stat.ValueAsInt * 0.01m).ToString(CultureInfo.InvariantCulture)
+                    : stat.ValueAsInt.ToString();
+                statText.text = string.Format(StatTextFormat, stat.Type, statValueText);
```

Comment:
These lines can be batter using extension method like below.

```
statText.text = stat.DecimalStatToString(); 
```

But `DecimalStatToString()` return with `+` character front of value. So it has to improved.